api/namex/VERSION.py

```diff
@@ -1 +1 @@
-__version__ = '1.1.38'
+__version__ = '1.1.40'
```

Comment:
Updated the version to 1.1.40 since https://github.com/bcgov/namex/pull/1490 updated the version to 1.1.39.